packages/manager/modules/pci/src/projects/project/storages/containers/add/translations/Messages_fr_FR.json

```diff
@@ -30,5 +31,22 @@
   "pci_projects_project_storages_containers_add_success_message": "Le conteneur {{container}} a été ajouté.",
   "pci_projects_project_storages_containers_add_success_message_with_user_creation": "Le conteneur {{container}} a été créé avec succès. L'utilisateur S3 {{ userName }} est administrateur de ce conteneur.",
   "pci_projects_project_storages_containers_add_error_post": "Une erreur est survenue lors de l'ajout du conteneur {{ container }} : {{ message }}.",
-  "pci_projects_project_storages_containers_add_type_price": "{{price}} HT/mois/Go"
+  "pci_projects_project_storages_containers_add_type_price": "{{price}} HT/mois/Go",
+  "pci_projects_project_storages_containers_deployment_mode_title": "Sélectionnez un mode de déploiement",
+  "pci_projects_project_storages_containers_add_deployment_mode_region-3-az": "Région 3-AZ",
+  "pci_projects_project_storages_containers_add_deployment_mode_region": "Région 1-AZ",
+  "pci_projects_project_storages_containers_add_deployment_mode_region-3-az_description": "Déploiement hautement résilient et disponible sur 3 zones de disponibilité.",
+  "pci_projects_project_storages_containers_add_deployment_mode_region_description": "Déploiement résilient et économique sur 1 zone de disponibilité.",
+  "pci_projects_project_storages_containers_add_deployment_mode_region-3-az_label": "3-AZ",
+  "pci_projects_project_storages_containers_add_deployment_mode_region_label": "1-AZ",
+  "pci_projects_project_storages_containers_add_deployment_mode_sub_title": "Sélectionnez la configuration optimale pour garantir la disponibilité, la résilience et la latence appropriée de vos données en fonction de vos cas d'usage.",
+  "pci_projects_project_storages_containers_add_deployment_mode_sub_title_link": "Consultez notre document ici",
+  "pci_projects_project_storages_containers_add_add_region_success": "La localisation de stockage {{ code }} a bien été ajoutée à votre projet Public Cloud.",
+  "pci_projects_project_storages_containers_add_add_region_error": "Une erreur est survenue lors de l'ajout de la localisation : {{message}}",
+  "pci_projects_project_storages_containers_add_add_region_activate": "La localisation de stockage sélectionnée n’est pas activée. Cliquez sur « Suivant » pour l’ajouter à votre projet Public Cloud et continuer la création de votre conteneur.",
+  "pci_projects_project_storages_containers_add_deployment_mode_soon": "Coming soon",
```

Comment:
No french translation here ?

Reply:
The business team wants it to be like that

Reply:
The business team wants it to be like that